### ISSUE TITLE
Does this library support Server Side Rendering (SSR)?

### ISSUE THREAD

**Issue by btakita** [Freak613/stage0#41]
I was not able to find anything indicating that this library supports SSR or rendering strings. Is it feasible to have SSR support on stage0?

**Comment by Freak613** [comment]
Yeah, this library was not designed to support SSR, it's targeted mainly to work in browser. Theoretically it can work in synthetic environments that support DOM API, like `jsdom`, however I'm afraid that it will not be as performant as string generation. 
Currently there are no plans to support SSR in this library, since it diverges with main point - to be a set of helper tools to work with DOM API.
